rioting-daisies/BangazonWorkforce
# Employee Details Functionality Implemented with ViewModel

# Description

This pull request is for the code that responsible for allowing the user to view the details of a particular employee. Once the Index view is created, the user can access this view by clicking the `Details` link next to an employee on the list. For now, we can access this feature by setting the proper route within our browser.

This pull request contains code written in the EmployeeController.cs file. There are two new methods: Details( ) and GetEmployeeById( ). I've also added the connection properties to the top.

Also within the pull request is the EmployeeDetailsViewModel.cs. This is the view model for the Employee Details view. This view model contains an Employee property, Computer property, Training Programs property, and some private methods and a constructor method. This is used to create a view model instance that will be passed in the the Employees/Details.cshtml view.


Fixes Issue # [(3)](https://github.com/rioting-daisies/BangazonWorkforce/issues/3)

## Type of change

Please delete options that are not relevant.



# Testing Instructions

1. `git fetch --all`
2. git checkout `blue-issueThree`
3. Look over `EmployeeController.cs` file and view the Details( ) and GetEmployeeById( ) methods.
4. Look over the `EmployeeDetailsViewModel.cs` file and review the code/comments.
5. Look over the `Views/Employees/Details.cshtml` file
6. Run the server
7. Put `http://localhost:5000/employees/details/1` into your browser to see the details.

# Checklist:
